test/wallet-test.js

```diff
@@ -1501,6 +1501,43 @@ describe('Wallet', function() {
     });
   }
 
+  for (const witness of [true, false]) {
+    it('should create witness multisig account for more than 15 keys', async () => {
+      const wallet = await wdb.create({
```

Comment:
I expect this to throw right away for legacy. Kinda surprised we don't already check excessive `n` size in `Account.fromOptions()`. Do you think we should? Maybe right after this:

https://github.com/bcoin-org/bcoin/blob/014a104899dcd9fafca1bcfea88203ed990a8e90/lib/wallet/account.js#L157-L158

Reply:
Oh yes I think we can add a check here as well